experimental/oak_functions_loader_linux_uds/src/main.rs

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use clap::Parser;
+use oak_remote_attestation::handshaker::{AttestationBehavior, EmptyAttestationVerifier};
+use oak_remote_attestation_amd::PlaceholderAmdAttestationGenerator;
+use std::os::unix::net::UnixStream;
+
+#[derive(Parser, Clone, Debug)]
+#[clap(about = "Oak Functions Loader Linux UDS")]
+pub struct Opt {
+    #[clap(long, help = "UDS address to use for the communication channel")]
+    pub comms_address: std::path::PathBuf,
+}
+
+struct Logger {}
```

Comment:
why not use some existing logger, such as `env_logger` we use for the launcher anyway?

Reply:
It seems desirable to disable logs entirely for release builds, to avoid inadvertent logs. That's the same approach the old linux binary runtime used: https://github.com/project-oak/oak/blob/main/oak_functions/loader/src/logger.rs